assets/contrastsheet.csv

```diff
@@ -0,0 +1,3 @@
+contrast,treatment,control
+GBR-YRI,GBR,YRI
+YRI-GBR,YRI,GBR
```

Comment:
I'm assuming this is genome dependent too, can that be reflected in the name as well?

Reply:
This was a file leftover from testing by accident. It is no needed anymore. Fixed in #68